### ISSUE TITLE
Get PRs out into supporting packages

### ISSUE THREAD

**Issue by shashi** [shashi/WebIO.jl#5]
- [ ] IJulia
- [ ] Mux
- [ ] Blink
- [ ] Atom

**Comment by ranjanan** [comment]
To depend on WebIO? Yes please. :-)

**Comment by ranjanan** [comment]
Yes, this would solve the Requires issues. In addition, it would help static compilation issues as well.

**Comment by rdeits** [comment]
Excellent! With that, #129 and #125 I think we could have v0.7 working.
